concent_api/concent_api/system_check.py

```diff
@@ -353,31 +353,31 @@ def create_error_50_middleman_port_is_not_set() -> Error:
 def create_error_51_middleman_port_has_wrong_type(value: Any) -> Error:
     return Error(
         f"Setting MIDDLEMAN_PORT has incorrect type `{type(value)}` instead of `int`.",
-        hint=f"Set setting MIDDLEMAN_PORT to be integer type.",
+        hint=f"Set setting MIDDLEMAN_PORT to be an integer.",
         id="concent.E051",
     )
 
 
 def create_error_52_middleman_port_has_wrong_value(value: int) -> Error:
     return Error(
         f"Setting MIDDLEMAN_PORT has incorrect value `{value}`.",
-        hint=f"Set setting MIDDLEMAN_PORT to be integer value between 0-65535.",
+        hint=f"Set setting MIDDLEMAN_PORT to be integer value between 0 and 65535.",
```

Comment:
Sorry, but it really troubles me: could you add `"0 and 65535 (exclusive)."`? https://www.quora.com/English-Grammar-Does-between-two-numbers-a-and-b-include-a-and-b-themselves

Reply:
Sure :+1: done